kncaputo/rancid-tomatillos
# Feature/stylings playground k

## What's this PR do?
- Add some styles to movie details page
   - Attempt to make it responsive
   - Center movie details and their descriptors
   - Add function to correctly display genres
- Add fetch for movie trailer
- Add `ReactPlayer` and movie trailer to movie details page

## Where should the reviewer start?
- Start by reviewing `MovieDetails`

## How should this be manually tested?
- test in browser by clicking a movie to see its details. Adjust screen size to see initial responsiveness 

## Any background context you want to provide?
- Definitely still a work in progress. Want to add a carousel to display movie trailers since many movies have multiple, and the backdrop image. Right now, trailer and image are stacked. 
- Movie trailers is an array and right now the trailer that's displayed is hardcoded at index 0. This throws an error for movies without trailers. We will address movies without trailers once we have the carousel.

## Screenshots/GIFs
<img width="677" alt="Screen Shot 2020-12-08 at 6 03 27 PM" src="https://user-images.githubusercontent.com/68157759/101551829-b4b01300-397f-11eb-8e98-e9aaf8ca6ff1.png">

<img width="536" alt="Screen Shot 2020-12-08 at 6 03 20 PM" src="https://user-images.githubusercontent.com/68157759/101551840-b8dc3080-397f-11eb-8644-57f67f6857ec.png">
